stories/Tables/Tables.stories.mdx

```diff
@@ -0,0 +1,150 @@
+</Canvas>
+
+Puoi anche invertire i colori, con testo chiaro su sfondi scuri con `.table-dark`.
+
+<Canvas>
+  <Story id='componenti-tables--table-dark' />
+</Canvas>
+
+## Opzioni delle intestazioni di tabella
+
+Simile alle tabelle e alle tabelle scure, utilizzare le classi di modifica `.thead-light` o `.thead-dark` per far apparire `<thead>` chiaro o grigio scuro.
+
+<Canvas>
+  <Story id='componenti-tables--table-thead' />
+</Canvas>
+
+## Righe striate
+
+Usa `.table-striped` per aggiungere delle striature zebrate ad ogni riga della tabella contenute in `<tbody>`.
+
+<Canvas>
+  <Story id='componenti-tables--table-striped' />
+</Canvas>
+
+## Tabella con bordi
+
+Aggiungi `.table-bordered` per avere i bordi a tutti i lati della tabella e su tutte le celle.
+
+<Canvas>
+  <Story id='componenti-tables--table-bordered' />
+</Canvas>
+
+## Tabella senza bordi
+
+Aggiungi la classe `.table-borderless` per una tabella senza bordi.
+
+<Canvas>
+  <Story id='componenti-tables--table-borderless' />
+</Canvas>
+
+## Righe ed hover
+
+Aggiungi `.table-hover` per abilitare lo stato hover sulle righe della tabella contenute in `<tbody>`.
+
+<Canvas>
+  <Story id='componenti-tables--table-hover' />
+</Canvas>
+
+## Tabella compatta
+
+Aggiungi `.table-sm` per rendere le tabelle più compatte dimezzando il cell padding.
+
+<Canvas>
+  <Story id='componenti-tables--table-compact' />
+</Canvas>
+
+## Classi contestuali
+
+Usa le classi contestuali per colorare le righe delle tabelle o le singole celle.
+
+<Canvas>
+  <Story id='componenti-tables--table-contextual' />
+</Canvas>
+
+<div className='docs'>
+  <Callout color='success'>
+    <CalloutTitle>
+      Trasmettere significato alle tecnologie assistive
+    </CalloutTitle>
+    <CalloutText>
+      L'uso del colore per aggiungere un significato fornisce solo
+      un’indicazione visiva, che non sarà trasmesso agli utenti di tecnologie
+      assistive – come gli screen reader. Assicurati che le informazioni
+      denotate dal colore siano rese disponibili anche dal contenuto stesso
+      (es.: il testo visibile), o siano incluse attraverso mezzi alternativi,
+      come testo aggiuntivo nascosto con la classe `.sr-only`.
+    </CalloutText>
+  </Callout>
+</div>
+
+Crea tabelle responsive racchiudendo ogni `.table` con `.table-responsive{-sm|-md|-lg|-xl}`, facendo scorrere orizzontalmente la tabella ad ogni breakpoint, rispettivamente con `max-width` di 576px, 768px, 992px, 1200px.
+
+<div className='docs'>
+  <Callout color='primary'>
+    <CalloutText>
+      Nota che in quanto i browser non supportano attualmente i range context
+      queries, stiamo aggirando i limiti dei prefissi min- and max- e viewports
+      con larghezze frazionarie (che possono verificarsi in determinate
+      condizioni su dispositivi ad alta risoluzione, ad esempio) utilizzando
+      valori con maggiore precisione per questi confronti.
+    </CalloutText>
+  </Callout>
+</div>
```

Comment:
I believe this content belongs to the `Responsive` section below. Opened an issue also on BI: https://github.com/italia/bootstrap-italia/issues/566

Reply:
I have removed paragraph with refs of responsive classname and moved the callout in correct section. What do you think?